checkProcesses.py

```diff
@@ -0,0 +1,31 @@
+import os
+import re
+import requests
+import subprocess
+
+def postMessage(message):
+    postMessageURL = "https://slack.com/api/chat.postMessage?token=" + os.environ['SLACK_USER_TOKEN_STRING'] + "&channel=G0PT4RSL9&as_user=true&link_names=1&text=" + message
```

Comment:
Any way we can have this Channel ID not be hardcoded? Might be nice, especially if we eventually want to make this public